packages/features/eventtypes/components/AddMembersWithSwitch.tsx

```diff
@@ -74,11 +74,11 @@ const CheckedHostField = ({
           }}
           value={(value || [])
             .filter(({ isFixed: _isFixed }) => isFixed === _isFixed)
-            .map((host) => {
+            .reduce((acc, host) => {
               const option = options.find((member) => member.value === host.userId.toString());
-              return option ? { ...option, priority: host.priority ?? 2, isFixed } : options[0];
-            })
-            .filter(Boolean)}
+              if (option) acc.push({ ...option, priority: host.priority ?? 2, isFixed });
```

Comment:
Early return is more readable, considering we could add more code in future here.

```suggestion
              if (!option){
                return acc
              }
              acc.push({ ...option, priority: host.priority ?? 2, isFixed });
```